managers/tests/test_unit_project_signals.py

```diff
@@ -33,3 +38,15 @@ def test_user_should_be_updated_to_employee_when_he_is_no_longer_manager_of_any_
         self.client.post(path=f"/admin/managers/project/{self.project.pk}/change/", args=self.project)
         self.manager.refresh_from_db()
         self.assertEqual(self.manager.user_type, CustomUser.UserType.EMPLOYEE.name)
+
+    def test_admin_should_not_be_updated_to_manager_when_he_is_assigne_as_a_project_manager(self):
+        self.project.managers.add(self.admin)
+        self.client.post(path="/admin/managers/project/add/", args=self.project)
+        self.admin.refresh_from_db()
+        self.assertNotEqual(self.admin.user_type, CustomUser.UserType.MANAGER.name)
```

Comment:
@Szymiks For future information. Always better to check if it is equal to something than not equal :) I'll fix it, because I want to merge it now :)